apps/src/codebridge/hooks/useInitialSources.ts

```diff
@@ -120,8 +120,12 @@ export const useInitialSources = (
     if (isStartMode) {
       return startSources;
     }
-    if (predictSettings?.isPredictLevel) {
-      // Predict levels never use sources loaded from the server, only the start sources.
+    if (
+      predictSettings?.isPredictLevel &&
+      predictSettings?.codeEditableAfterSubmit === false
```

Comment:
Instead of `predictSettings?.codeEditableAfterSubmit === false` can we use `!predictSettings.codeEditableAfterSubmit`? Not sure if there's a reason that falsy values can't be used.

Reply:
I think we could use either. When predict settings are added to a level, we prevent any empty values being saved so `false` is the only falsy value we should expect here. I think it comes down to preference.

IMHO, we're making a decision that significantly changes behavior (return early, load different sources), and false here is meaningful. It's not just missing a value; it's confirmation that the user will never be able to edit their code.